src/react/components/StyleEditor.jsx

```diff
@@ -22,7 +22,7 @@ const TabSelector = styled(Radio.Group)`
 `;
 
 const StaticForm = styled(Form)`
-    width: 400px;
+    width: 522px;
```

Comment:
Do we need both: the pixel width here AND `width={ 620 }` in AdminLayerForm/VisualizationTabPane/VectorStyle.jsx? Could we make it work with one or the other?

Reply:
Removed this.